Runtime/UrdfComponents/UrdfJoints/UrdfJointPrismatic.cs

```diff
@@ -8,8 +8,14 @@ namespace RosSharp.Urdf
 {
     public class UrdfJointPrismatic : UrdfJoint
     {
+#if !UNITY_2020_1_OR_NEWER
+#error A Unity version of 2020.1 or newer is required to use UrdfJointPrismatic.cs
+#endif
```

Comment:
This should not be required here; it's preferable to allow the package manager to enforce version constraints using the [package.json](https://github.com/Unity-Technologies/URDF-Importer/blob/main/package.json#L6-L7).

Reply:
> This should not be required here; it's preferable to allow the package manager to enforce version constraints using the [package.json](https://github.com/Unity-Technologies/URDF-Importer/blob/main/package.json#L6-L7).

Noted. I will remove these lines.